src/powershell/install.sh

```diff
@@ -162,6 +163,12 @@ if [ ${#POWERSHELL_MODULES[@]} -gt 0 ]; then
     done
 fi
 
+# If URL for powershell profile is provided, download it to '/opt/microsoft/powershell/7/profile.ps1'
+if [ -n "$POWERSHELL_PROFILE_URL" ]; then
+    echo "Downloading PowerShell Profile from: $POWERSHELL_PROFILE_URL"
+    curl -sSL -o "/opt/microsoft/powershell/7/profile.ps1" "$POWERSHELL_PROFILE_URL"
+fi
```

Comment:
Also, can you help me understand your motivation behind this change? I have never used Powershell profiles before, and I don't have my own script (hence, this question). 

Do folks like to publish their setting to some location? or are there official scripts available which you like to use?

Reply:
My PowerShell profile holds a lot of customization (e.g prompt to show git status, custom aliases and functions) and useful utilities like [Completion Predictor](https://github.com/PowerShell/CompletionPredictor) and tab auto completer for lots of tools.

Folks usually publish their profiles to Github. So there are no official scripts etc, just personal profiles usually from Github but could be anywhere on a webserver as long as its publicly accessible.